01_fib/joel00wood/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func fibCalc(n float64) float64 {
+	switch n {
+	case 0:
+		return n
+	case 1:
+		return n
+	default:
+		return fibCalc(n-1) + fibCalc(n-2)
```

Comment:
I had the same compulsion to do it recursively, however as a thought experiment it's worth considering what the "nicest" way to do it non-recursively is